Fix deploy (#326)
wso2/docs-mi: en/docs/develop/deploy-artifacts.md

# Deploying Artifacts

> Now you have developed an integration using the Micro Integrator VS Code plugin, we have several options to deploy that integration to the Micro Integrator server runtime.

Reviewer: ```suggestion
Now that you have developed an integration using the Micro Integrator Visual Studio Code plugin, you have several options to deploy that integration to the Micro Integrator server runtime.
```

## Build and Run

> 1.	[Download and install]({{base_path}}/install-and-setup/install/installing-mi) the Micro Integrator server and on your computer.

Reviewer: ```suggestion
1.	[Download and install]({{base_path}}/install-and-setup/install/installing-mi) the Micro Integrator server on your local machine.
```

2.  Add the server to the VS Code extension using the following steps (if not already).
    1. Open the command palette by pressing `Ctrl+Shift+P` and select the command `Add MI Server`.
    2. In the file selector that opens, select the Micro Integrator server directory.
       <a href="{{base_path}}/assets/img/develop/add-server.png"><img src="{{base_path}}/assets/img/develop/add-server.png" alt="addServer" width="700"></a>
    

> 3.  Bring the **Project Overview** page by clicking on the **Project Overview** icon.

Reviewer: ```suggestion
3.  Open the **Project Overview** page by clicking on the **Project Overview** icon.
```

- To build and export in single step, click on the **Export** button on top right corner of the **Project Overview** page to export the Carbon Application.
  Select a directory to save the exported Carbon Application file.

    <a href="{{base_path}}/assets/img/develop/export-capp.png"><img src="{{base_path}}/assets/img/develop/export-capp.png" alt="export" width="700"></a>

Once the Carbon Application is exported, we can copy it to the `<MI_HOME>/repository/deployment/server/carbonapps` directory manually and start the server.

## Build Docker image

We can use the Micro Integrator VS Code extension to build a Docker image of our integration solution.

1.  Make sure the Docker is installed in your computer and the Docker daemon is running.
2.  Click on the **Build** button on top right corner of the **Project Overview** page and select docker option to build the Docker image.

    <a href="{{base_path}}/assets/img/develop/build-docker.png"><img src="{{base_path}}/assets/img/develop/build-docker.png" alt="dokcer" width="700"></a>

3. Run `docker images` command to verify the Docker image built correctly.
4. Run the Docker image using the following command.

    ```bash

>     docker run -p 8290:8290 -p 8253:8253 -p 9164:9164 <image-name>

Reviewer: ```suggestion
    docker run -p 8290:8290 -p 8253:8253 -p 9164:9164 <DOCKER_IMAGE-NAME>
```

>     docker run -p 8290:8290 -p 8253:8253 -p 9164:9164 <DOCKER_IMAGE-NAME>

Reviewer: ```suggestion
    docker run -p 8290:8290 -p 8253:8253 -p 9164:9164 <DOCKER_IMAGE_NAME>
```

    ```

>     Replace `<image-name>` with the name of the Docker image you have built.

Reviewer: ```suggestion
    Replace `<DOCKER_IMAGE_NAME>` with the name of the Docker image you have built.
```
